### ISSUE TITLE
ModuleNotFoundError: No module named 'pyaf.ForecastEngine'

### ISSUE THREAD

**Issue by eoagustin** [antoinecarme/pyaf#150]
Hi,

Im encountering module not found error when running the demo python script. I imported the git through:

!pip3 install --upgrade git+https://github.com/antoinecarme/pyaf.git

When I tried looking at the submodules inside the module through git, this is what was returned:

from github_com.antoinecarme import pyaf
def sort_by_type(t):
    return t[0].__name__
attrs = [(type(getattr(pyaf, attr)), attr) for attr in dir(pyaf)]
attrs.sort(key=sort_by_type)
for t, n in attrs:
    print(t, n)

<class '_frozen_importlib.ModuleSpec'> __spec__
<class 'NoneType'> __doc__
<class '_frozen_importlib_external.SourceFileLoader'> __loader__
<class 'dict'> __builtins__
<class 'list'> __path__
<class 'str'> __cached__
<class 'str'> __file__
<class 'str'> __name__
<class 'str'> __package__ 

I also tried installing through pip but to no use

**Comment by eoagustin** [comment]
I also tried answers from #123 but to no use

**Comment by eoagustin** [comment]
Please use this instead:

from github_com.antoinecarme import pyaf

def sort_by_type(t):
    return t[0].__name__

attrs = [(type(getattr(pyaf, attr)), attr) for attr in dir(pyaf)]
attrs.sort(key=sort_by_type)
for t, n in attrs:
    print(t, n)

**Comment by antoinecarme** [comment]
@eoagustin 

Thanks for using PyAF.

I do not understand why you use : 

```
from github_com.antoinecarme import pyaf
```


Isn't a simple 
```
import pyaf
```
enough ?

**Comment by eoagustin** [comment]
Hello. Yes I was able to import it successfully. However, I encountered another issue and I dont know if I should post it here and edit the issue title and description instead. 

Anyway, issue is that my subprocess.Popen was not able to find my dot.exe from graphviz. Im using cloudera data science workbench. I installed graphviz through conda (local) since its not permitted in our group to use sudo apt install graphviz (which was mentioned as solution in other issues in this git). I located the directory of the dot.exe in my local and added it in my os.environ("PATH") so that I can use it in cds workbench. However, it wasnt able to find the executable in the said path. 

pydot==1.4.1
graphviz==0.15
pyaf==2.0.1
Python 2.7.18
